subsys/bluetooth/shell/bt.c

```diff
@@ -2929,6 +2958,51 @@ static int cmd_set_power_report_enable(const struct shell *sh, size_t argc, char
 
 #endif
 
+#if defined(CONFIG_BT_PATH_LOSS_MONITORING)
+static int cmd_set_path_loss_reporting_parameters(const struct shell *sh, size_t argc, char *argv[])
+{
+	if (argc != 6) {
+		shell_help(sh);
+		return SHELL_CMD_HELP_PRINTED;
+	}
+	if (default_conn == NULL) {
+		shell_error(sh, "Conn handle error, at least one connection is required.");
+		return -ENOEXEC;
+	}
+	const struct bt_conn_le_path_loss_reporting_parameters params = {
+		.high_threshold = strtoul(argv[1], NULL, 10),
+		.high_hysteresis = strtoul(argv[2], NULL, 10),
+		.low_threshold = strtoul(argv[3], NULL, 10),
+		.low_hysteresis = strtoul(argv[4], NULL, 10),
+		.min_time_spent = strtoul(argv[5], NULL, 10),
+	};
+
+	int err = bt_conn_le_set_path_loss_monitoring_parameters(default_conn,
+								 &params);
+	return err;
+}
+
+static int cmd_set_path_loss_reporting_enable(const struct shell *sh, size_t argc, char *argv[])
+{
+	if (argc != 2) {
```

Comment:
Are these `argc` checks needed? I thought the `SHELL_CMD_ARG` macro defined the number of required and optional arguments.